library/src/blas2/rocblas_sbmv.cpp

```diff
@@ -0,0 +1,173 @@
+                            "incy",
+                            incy);
+        }
+
+        if(uplo != rocblas_fill_lower && uplo != rocblas_fill_upper)
+            return rocblas_status_invalid_value;
+
+        if(n < 0 || k < 0 || lda < n || lda < 1 || !incx || !incy)
+            return rocblas_status_invalid_size;
+
+        // quick return before pointer checks
+        if(!n)
+            return rocblas_status_success;
+
+        if(!A || !x || !y || !alpha || !beta)
+            return rocblas_status_invalid_pointer;
+
+        return rocblas_sbmv_template<T>(
+            handle, uplo, n, k, alpha, 0, A, 0, lda, 0, x, 0, incx, 0, beta, 0, y, 0, incy, 0, 1);
+    }
+
+} // namespace
+
+/*
+* ===========================================================================
+*    C wrapper
+* ===========================================================================
+*/
+
+extern "C" {
+
+#ifdef IMPL
+#error IMPL ALREADY DEFINED
+#endif
+
+#define IMPL(routine_name_, T_)                                                                  \
+    rocblas_status routine_name_(rocblas_handle handle,                                          \
+                                 rocblas_fill   uplo,                                            \
+                                 rocblas_int    n,                                               \
+                                 rocblas_int    k,                                               \
+                                 const T_*      alpha,                                           \
+                                 const T_*      A,                                               \
+                                 rocblas_int    lda,                                             \
+                                 const T_*      x,                                               \
+                                 rocblas_int    incx,                                            \
+                                 const T_*      beta,                                            \
+                                 T_*            y,                                               \
+                                 rocblas_int    incy)                                            \
+    try                                                                                          \
+    {                                                                                            \
+        return rocblas_sbmv_impl<T_>(handle, uplo, n, k, alpha, A, lda, x, incx, beta, y, incy); \
+    }                                                                                            \
+    catch(...)                                                                                   \
+    {                                                                                            \
+        return exception_to_rocblas_status();                                                    \
+    }
+
+IMPL(rocblas_ssbmv, float);
```

Comment:
I see that cuBLAS only supports float/double for sbmv and spmv, however LAPACK does in fact support complex. Not sure if we should also be supporting them or not.

Reply:
Matched blas and cublas which is SD only.  most complex symmetrics are hermitian complex I presumed so hbmv, hpmv.   Added CZ to symv to match cublas.  Can add them in a different PR if there is interest, maybe Juan would like them.